pkg/transcriptiondata/eurasian.tsv

```diff
@@ -146,27 +146,27 @@ a˞ː	long rhotacized unrounded open front vowel	+	+	a˞ː	http://eurasianphonol
-ãũ	from nasalized unrounded open front to nasalized rounded close back diphthong	+	+	ãũ	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83u%CC%83					2	
-ãu̯	from nasalized unrounded open front to non-syllabic rounded close back diphthong	+	+	ãw	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83w					1	
-ãũ̯	from nasalized unrounded open front to non-syllabic nasalized rounded close back diphthong	+	+	ãw̃	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83w%CC%83					1	
-ãɛ̃	from nasalized unrounded open front to nasalized unrounded open-mid front diphthong	+	+	ãɛ̃	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83%C9%9B%CC%83					1	
-ãɪ̯ː	from nasalized unrounded open front to long non-syllabic unrounded near-close near-front diphthong	+	+	ãɪ̯ː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83%C9%AA%CC%AF%CB%90					1	
-ãː	long nasalized unrounded open front vowel		+	ãː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83%CB%90					19	
-ã̤	nasalized breathy unrounded open front vowel	+	+	ã̤	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83%CC%A4					1	
-ä	centralized unrounded open front vowel		+	ä	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88					73	
-äi̯	from centralized unrounded open front to non-syllabic unrounded close front diphthong	+	+	äi̯	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88i%CC%AF					3	
-äu̯	from centralized unrounded open front to non-syllabic rounded close back diphthong	+	+	äu̯	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88u%CC%AF					5	
-äː	long centralized unrounded open front vowel		+	äː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CB%90					39	
-ãː̈	long centralized nasalized unrounded open front vowel	+	+	äː̃	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CB%90%CC%83					1	
-ã̈	centralized nasalized unrounded open front vowel	+	+	ä̃	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%83					8	
-ãː̈	long centralized nasalized unrounded open front vowel	+	+	ä̃ː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%83%CB%90					4	
-ã̤ː̈	long centralized nasalized breathy unrounded open front vowel	+	+	ä̤̃ː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%83%CC%A4%CB%90					1	
-ä	centralized unrounded open front vowel		+	ä̠	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%A0					4	
-ä̤	centralized breathy unrounded open front vowel	+	+	ä̤	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%A4					1	
+ã	nasalized unrounded open front vowel		+	ã	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%83					45	

```

Comment:
@cormacanderson, this is just a note for your interest: all ã and other nasal instances are listed here as modified from the original, although you apparently did not touch them. Why so? Because I assume you edited all in one file as TSV, and then pasted it here, right? What happens is that spreadsheet systems often choose a direct solution on whether to code ã as one unicode symbol or two. Luckily, CLTS uses a strict normalization towards the two-symbol solution, so the data here can be submitted in any form as we will modify it later anyway, when checking. However, it shows how tedious working with Unicode can be.